src/Compilers/Test/Core/TargetFrameworkUtil.cs

```diff
@@ -32,7 +32,7 @@ public enum TargetFramework
         NetStandard20,
         NetCoreApp,
         NetFramework,
-        StandardLatest,
+        NetLatest,
```

Comment:
The reason for this change is everywhere else in the code the term `Standard` meant "subset of real assemblies". As we moved to the real `net461` reference assemblies I moved it to be the full set of assemblies (as it is for all the other core TF). Hence I normalized the names at the same time. 

Eventually I'd like to look at removing the `Standard` variety but that is a longer tail item.

Reply:
Let's add a comment (this is "latest" on either desktop or Core)